app/assets/javascripts/application.js

```diff
@@ -451,6 +451,17 @@ function setPowerState(item, status){
   $('[rel="twipsy"]').tooltip();
 }
 
+function disable_textfield(item) {
```

Comment:
name is misleading, afaiu it also enable disabled text fields? (or any input for that matter?)